prestoadmin/slider/config.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+Module for setting and validating the presto-admin slider config
+"""
+
+from functools import wraps
+
+from fabric.context_managers import settings
+from fabric.state import env
+from fabric.operations import prompt
+
+from prestoadmin import config
+from prestoadmin.config import ConfigFileNotFoundError
+
+from prestoadmin.util.validators import validate_host, validate_port, \
+    validate_username, validate_can_connect, validate_can_sudo
+
+SLIDER_CONFIG_LOADED = 'slider_config_loaded'
+SLIDER_CONFIG_PATH = '/etc/opt/prestoadmin/slider/config.json'
```

Comment:
I've fixed my usage here, and I'll put up a separate PR to fix it globally in the tests.